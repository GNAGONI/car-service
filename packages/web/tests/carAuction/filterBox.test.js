import React from 'react'
import renderer from 'react-test-renderer'
import FilterBox from '../../components/carAuction/filterBox'

it('renders correctly', () => {
  const tree = renderer.create(<FilterBox />).toJSON()
  expect(tree).toMatchSnapshot()
})
