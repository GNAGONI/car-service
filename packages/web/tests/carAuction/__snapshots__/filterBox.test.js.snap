// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<article
  className="col-12 col-md-4 box-filter text-center"
>
  <div
    className="holder"
  >
    <div
      className="img-area"
    >
      <img />
    </div>
    <div
      className="text-box"
    >
      <h2>
        <strong>
          Browse by 
          <br />
        </strong>
      </h2>
      <div
        className="btn-block"
      >
        <button
          className="btn btn-primary btn-ml"
          onClick={[Function]}
        >
          Search
        </button>
      </div>
    </div>
  </div>
</article>
`;
